change/@ni-nimble-components-9b50b032-00a9-4a25-9089-308c223068ad.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Now that I understand the state of the spinner better I realized you do have 2 already in the spinner mapping. If it's not too messy I think it would help to have both spinner and icon examples with both text hidden and not hidden. That's getting a bit like a matrix test which isn't ideal, but I think it's useful since we can't have controls on the mapping args table.

Reply:
> If it's not too messy I think it would help to have both spinner and icon examples with both text hidden and not hidden.

Technically, the "Unknown" status of Abbey has `text-hidden` set on it. Though, I can see why that wouldn't be extremely clear since it also doesn't use an icon. I'm open to suggestions on how to make the storybook example clearer, but I'm concerned about doing too much to show every combination of configurations since I don't think we want it to deviate too far from whatever our best practices will end up being.

Reply:
That's fair, I guess I was optimizing for what I want as a reviewer, not what would be most useful to a client. I agree that the latter is more important for the docs in the mdx. For the combinations that aren't best practices we could either have a separate client-visible story that demos them (but doesn't appear in the mdx) or reviewers can see them in the matrix story.

I would be fine to leave this as-is and revisit it when we've figured out our best practices.